definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-referral-nonprod.json

```diff
@@ -109,6 +109,18 @@
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Yes"
   },
+  {
+    "LiveFrom": "02/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "FeeAmountWithoutNotice",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "GeneralReferralPayment1",
+    "PageLabel": "General referral payment",
+    "PageDisplayOrder": 1,
+    "FieldShowCondition": "GeneralReferralPaymentType=\"hiddenfield\""
+  },
```

Comment:
@jacmurphy Does this change relate to the ticket? Looked through Jira but didn't find or is this some another change made in here?

Reply:
Yep this can be removed now :)

Reply:
Good eye @madjava ! i was planning on removing it before anyone noticed but youre too good!